ocs_ci/helpers/dr_helpers_ui.py

```
@@ -200,6 +204,159 @@ def verify_drpolicy_ui(acm_obj, scheduling_interval):
     )
 
 
+def verify_pending_cleanup_alert_firing(acm_obj, scheduling_interval, operation="Failover"):
```

Ruff (0.15.14)

[warning] Unused function argument: `scheduling_interval`

(ARG001)